python/ray/data/_internal/execution/streaming_executor.py

```diff
@@ -411,13 +415,16 @@ def walk(op):
         raise ValueError(error_message.strip())
 
 
-def _debug_dump_topology(topology: Topology) -> None:
+def _debug_dump_topology(topology: Topology, log_to_stdout: bool = True) -> None:
     """Print out current execution state for the topology for debugging.
 
     Args:
         topology: The topology to debug.
     """
-    logger.get_logger().info("vvv scheduling trace vvv")
+    logger.get_logger(log_to_stdout).info("Scheduling Trace:")
```

Comment:
nit, I think the previous log message (something like "execution progress") is more understandable.